README.md

```diff
@@ -244,7 +245,17 @@ Updates to the above configurations require a reboot of all containers to take e
 > $ docker compose -f docker/docker-compose.yml up -d
 > ```
 
-## 🔧 Build a Docker image without embedding models
+### Switch ElasticSearch to Infinity as the chunk store
+
+RAGFlow uses ElasticSearch as the chunk store(fulltext and vector) by default. If you want to switch to [Infinity](https://github.com/infiniflow/infinity/), follow the steps below:
```

Comment:
RAGFlow uses Elasticsearch full-text and vector store by default.

Reply:
done